go-for-eat/components/Map/Map.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import s from './styles';
+
+class EventList extends Component {
```

Comment:
Naming: if it's going to be the map call it map, if it's going to be the eventlist call the file eventlist.